src/style/app.module.css

```diff
@@ -548,9 +603,8 @@
 .inputField {
   margin-top: 10px;
   margin-bottom: 10px;
-  background-color: var(--bs-white);
-  border: none;
-  box-shadow: 0 1px 1px var(--input-shadow-color);
+  background-color: var(--white-color);
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`